app/Response/Export.hs

```diff
@@ -0,0 +1,42 @@
+{-# LANGUAGE OverloadedStrings #-}
+
+module Response.Export
+    (exportGraphResponse, exportTimetableResponse) where
+
+import Control.Monad.IO.Class  (liftIO)
+import qualified Data.Map as M
+import Happstack.Server
+import ImageConversion
+import Database.Persist.Sql (toSqlKey)
+import Data.Int(Int64)
+
+-- | Serves a pdf file of a graph image
+exportGraphResponse :: ServerPart Response
+exportGraphResponse = do
+    req <- askRq
+    let cookies = M.fromList $ rqCookies req
+        gId = maybe "1" cookieValue (M.lookup "active-graph" cookies)
```

Comment:
This is out of date. Make sure you have the latest version of master, and look at the `graphImageResponse` function in `Image.hs`.